dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -1252,30 +1255,37 @@ public boolean isParentPermissionable() {
 	 * @return
 	 */
 	@JsonIgnore
-    public static Object lazyMetadataLoad ( String inode, String structureInode ) {
+    public static Object lazyMetadataLoad (final String inode, final String structureInode) {
```

Comment:
This method makes no sense in the Contentlet.java and should be removed - we know the content type and we know the inode.  What calls this? It seems like an internal method?

We should also strip out any references to the old Structures and Field when we come across them in code.